storage/remote/client.go

```diff
@@ -83,6 +83,7 @@ func (c *Client) Store(ctx context.Context, req *prompb.WriteRequest) error {
 		return err
 	}
 	httpReq.Header.Add("Content-Encoding", "snappy")
+	httpReq.Header.Add("Accept-Encoding", "snappy")
```

Comment:
This is the write request, the read request is below.

Reply:
Urgh, so much for making quick changes in the GitHub UI. 
Fixed.